website/client/components/inventory/equipment.vue

```diff
@@ -151,7 +162,9 @@ export default {
   },
   methods: {
     equip (item) {
-      this.$store.dispatch('common:equip', {key: item.key, type: this.costume ? 'costume' : 'equipped'});
+      if (!this.costume || this.user.preferences.costume) {
```

Comment:
the starts are still going to show up with this I suppose, just not working. Maybe we should not show them at all?